lib/cog/commands/role.ex

```diff
@@ -1,18 +1,16 @@
 defmodule Cog.Commands.Role do
   @moduledoc """
-  This command allows the user to manage roles. Roles can contain users.
+  This command allows the user to manage roles.
 
   Usage:
       role --create <rolename>
       role --drop <rolename>
-      role --grant [--user=<username>|--group=<groupname>] <rolename>
-      role --revoke [--user=<username>|--group=<groupname>]  <rolename>
+      role --grant --group=<groupname> <rolename>
```

Comment:
Since you can only grant roles to groups would it better syntax if group was an arg?

`role --grant <groupname> <rolename>`

Or is that something to revisit later?